setup.py

```diff
@@ -17,6 +17,5 @@
     ],
     packages=['aiotask_context'],
     install_requires=install_requires,
-    setup_requires=['pytest-runner'],
```

Comment:
@argaen do you remember why we were asking for this dependency in the `setup_requires` step?

Reply:
I think it is so you can do something like `python setup.py tests` or something like that. But happy to remove yeah